src/main.c

```diff
@@ -328,7 +328,13 @@ static uint32_t softdev_init(bool init_softdevice)
       .accuracy     = NRF_CLOCK_LF_ACCURACY_250_PPM
   };
 
+#ifdef BLE
```

Comment:
I don't like to have an extra macro for this, please use the ANT_LICENSE_KEY for this as well. These macro should be removed from the makefile as well.

```
ifndef ANT_LICENSE_KEY
```